Could not resolve test route

### Have you experienced this bug with the latest version of the template?

Yes

### Steps to Reproduce

1. `npx create-remix --template hyper63/alternative-stack`
2. `npm run dev`


### Expected Behavior

Dev server able to start

### Actual Behavior

```
npm run dev
...
Watching Remix app in development mode...
⚠️ Test routes enabled.
[WARN] Using experimental features. These features can change at any time.
{ ok: false, msg: "cache already exists", status: 409 }
CACHE:CREATE_STORE: {"date":"2022-09-02T16:40:30.819Z","ok":false,"status":409,"msg":"cache already exists"}
{ INFO: loaded databases: [notes-dev], DATE: 2022-09-02T16:40:30.832Z }
{ ok: false, msg: "database already exists", status: 409 }
DATA:CREATE_DB: {"date":"2022-09-02T16:40:30.837Z","ok":false,"status":409,"msg":"database already exists"}
[INFO] Created cache ⭐️ service in notes-dev domain ⚡️
[INFO] Created data 💿 service in notes-dev domain ⚡️

✘ [ERROR] Could not resolve ".//Users/maxim/src/cashflow-companion/test/test-routes/delete-user.ts"

    browser-route-module:/Users/maxim/src/cashflow-companion/test/test-routes/delete-user.ts?browser:1:24:
      1 │ ...ult } from ".//Users/maxim/src/cashflow-companion/test/test-rout...
        ╵               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Build failed with 1 error:
browser-route-module:/Users/maxim/src/cashflow-companion/test/test-routes/delete-user.ts?browser:1:24: ERROR: Could not resolve ".//Users/maxim/src/cashflow-companion/test/test-routes/delete-user.ts"
💿 Built in 564ms
```

The file is present and for some reason remix is not able to find it.

--- a/app/services/note.server.ts
+++ b/app/services/note.server.ts
@@ -37,10 +37,17 @@ export const NotesServerFactory = (env: ServerContext): NoteServer => {
     }
 
     // TODO: use hyper cache to instead of querying db
-    const { docs } = await hyper.data.query<NoteDoc>({
+    const res = await hyper.data.query<NoteDoc>({
       type: "note",
       parent,
     });
+
+    if (!res.ok) {
+      throw new Error(res.msg);
+    }
+
+    const { docs } = res;
+
     return docs.map(fromHyper.as(NoteSchema));
   }
 

--- a/app/services/user.server.ts
+++ b/app/services/user.server.ts
@@ -30,7 +30,13 @@ export const UserServerFactory = (env: ServerContext): UserServer => {
     const { hyper } = env;
 
     email = EmailSchema.parse(email);
-    const { docs } = await hyper.data.query<UserDoc>({ type: "user", email });
+    const res = await hyper.data.query<UserDoc>({ type: "user", email });
+
+    if (!res.ok) {
+      throw new Error(res.msg);
+    }
+
+    const { docs } = res;
 
     if (!docs.length) {
       return null;
@@ -53,10 +59,16 @@ export const UserServerFactory = (env: ServerContext): UserServer => {
       throw new NotFoundError(`user with email ${email} not found`);
     }
 
-    const { docs } = await hyper.data.query<PasswordDoc>({
+    const res = await hyper.data.query<PasswordDoc>({
       type: "password",
       parent: user.id,
     });
+
+    if (!res.ok) {
+      throw new Error(res.msg);
+    }
+
+    const { docs } = res;
 
     if (!docs.length) {
       // TODO: use hyper queue to create job to send alert

--- a/package.json
+++ b/package.json
@@ -51,6 +51,7 @@
     "@types/react": "^18.0.9",
     "@types/react-dom": "^18.0.4",
     "@vitejs/plugin-react": "^1.3.2",
+    "@vitest/coverage-c8": "^0.23.2",
     "c8": "^7.11.3",
     "cross-env": "^7.0.3",
     "cypress": "^9.7.0",
@@ -66,8 +67,8 @@
     "tailwindcss": "^3.0.24",
     "ts-node": "^10.8.0",
     "typescript": "^4.6.4",
-    "vite-tsconfig-paths": "^3.4.1",
-    "vitest": "^0.12.9"
+    "vite-tsconfig-paths": "^3.5.0",
+    "vitest": "^0.23.2"
   },
   "engines": {
     "node": ">=14"

--- a/remix.config.js
+++ b/remix.config.js
@@ -16,7 +16,9 @@ module.exports = {
       if (process.env.NODE_ENV === "production") return;
 
       console.log("⚠️ Test routes enabled.");
-      route("__tests/delete-user", path.join(__dirname, "test/test-routes/delete-user.ts"));
+
+      const appDir = path.join(__dirname, "app");
+      route("__tests/delete-user", path.relative(appDir, "test/test-routes/delete-user.ts"));
     });
   },
 };

--- a/test/setup-test-env.ts
+++ b/test/setup-test-env.ts
@@ -1,4 +1,4 @@
-import { installGlobals } from "@remix-run/node/globals";
+import { installGlobals } from "@remix-run/node";
 import "@testing-library/jest-dom/extend-expect";
 
 installGlobals();

